### PR TITLE
Authenticate Mattermost against Nagios instance

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Allow admins to configure basic HTTP authentication credentials for authentication against the Nagios instance.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-nagios/issues/42

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -25,6 +25,18 @@
                 "type": "text",
                 "help_text": "The URL for your Nagios instance. Must start with http:// or https://."
             },
+            {
+                "key": "NagiosUsername",
+                "display_name": "Nagios Username",
+                "type": "text",
+                "help_text": "The Nagios user used to authentiacte against your Nagios instance."
```
Comment (mickmister) [inline plugin.json:32]:
```suggestion
                "help_text": "The Nagios user used to authenticate against your Nagios instance."
```

**Review by mickmister — APPROVED** [review]
LGTM Nice work!

**Comment by hanzei** [comment]
@DHaussermann The PR is ready for QA review

**Review by javaguirre — APPROVED** [review]
LGTM

**Pull request merged**